packages/perseus/src/widgets/interactive-graphs/locked-figures/locked-function.tsx

```diff
@@ -0,0 +1,30 @@
+import * as KAS from "@khanacademy/kas";
+import {Plot} from "mafs";
+import * as React from "react";
+
+import {lockedFigureColors} from "../../../perseus-types";
+
+import type {LockedFunctionType} from "../../../perseus-types";
+
+const LockedFunction = (props: LockedFunctionType) => {
+    const {color, strokeStyle, directionalAxis, domain} = props;
+    const plotProps = {
+        color: lockedFigureColors[color],
+        style: strokeStyle,
+        domain,
+    };
+    const equation = props.equationParsed || KAS.parse(props.equation).expr;
```

Comment:
When is `equationParsed` populated?

Reply:
I was initially thinking that it would be populated when the content editor types the equation. I'm not sure how that would persist, though, so I'm rethinking the timing/management of this.